.github/workflows/inspektor-trigger-e2e-tests.yaml

```diff
@@ -0,0 +1,14 @@
+name: inspektor-e2e-tests
```

Comment:
We should not push this to main

Reply:
@rcohencyberarmor ?

Reply:
I added this for the inspektor branch...